src/core/Microsoft.PowerToys.Settings.UI/loc/it/src/core/Microsoft.PowerToys.Settings.UI/Strings/en-us/Resources.resw.lcl

```diff
@@ -1495,6 +1495,33 @@
         </Str>
         <Disp Icon="Str" />
       </Item>
+      <Item ItemId=";PowerLauncher_Radio_Theme_Dark.Content" ItemType="0;.resx" PsrId="211" Leaf="true">
+        <Str Cat="Text">
+          <Val><![CDATA[Dark]]></Val>
+          <Tgt Cat="Text" Stat="Loc" Orig="New">
+            <Val><![CDATA[Scuro]]></Val>
+          </Tgt>
+        </Str>
+        <Disp Icon="Str" />
+      </Item>
+      <Item ItemId=";PowerLauncher_Radio_Theme_Default.Content" ItemType="0;.resx" PsrId="211" Leaf="true">
+        <Str Cat="Text">
+          <Val><![CDATA[System default]]></Val>
+          <Tgt Cat="Text" Stat="Loc" Orig="New">
+            <Val><![CDATA[Impostazione predefinita del sistema]]></Val>
```

Comment:
This should be: "Impostazione predefinita di sistema"

Reply:
Why is it wrong to use a _preposizione articolata_ here?